.github/ISSUE_TEMPLATE/usage_question.yaml

```diff
@@ -0,0 +1,66 @@
+name: Usage Question
+description: Ask a question
+labels: ["Type: usage"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        While we enable issues as a mechanism for new contributors and passers-by who are unfamiliar with Apache Software Foundation projects to ask questions and interact with the project, we encourage users to ask such questions on public mailing lists:
```

Comment:
Nit, but it would be nice to ensure text files are word-wrapped at not more than 90 characters per line.

Reply:
I did so originally and found that the rendering of YAML injects line breaks, causing it to render very poorly on some displays. For example, on smaller widths displays like mobile devices, the line would soft wrap before 90 characters, then hard return after just two words on the next line, in the middle of a sentence. I'm happy to explore how to represent this best in source without those problems, but elected to remove line breaks in the short term.

Reply:
Hmm, apparently if you replace the `|` character with `>`, those newlines will be converted into spaces:
https://stackoverflow.com/a/21699210/10194
https://yaml-multiline.info/

Reply:
Incorporated into https://github.com/apache/arrow/pull/14688.